### PR TITLE
docs: improve documentation about configuration

### PR DESCRIPTION
**Pull request by gaetanars**
Improve documentation about configuration.

Remove Quickstart and add more arguments in Configuration.

### CODE REVIEW

File: docs/Configuration.md [hunk 1]
```diff
@@ -90,7 +54,7 @@ Here is an example key file:
 Use the JSON file to generate a Kubernetes secret.
 
 ```shell
-kubectl create secret generic fe-creds --from-file=credentials=./<JSON file name>
+kubectl -n upbound-system create secret generic fe-creds --from-file=credentials=./<JSON file name>
```
Comment (azrod) [inline docs/Configuration.md:57]: In all examples we are use `crossplane-system`

**Review by azrod — COMMENTED** [review]
see my comment

File: docs/Configuration.md [hunk 2]
```diff
@@ -114,3 +78,41 @@ spec:
 ```
```
Comment (azrod) [inline docs/Configuration.md:79]: Shouldn't we add a table of possible and required fields and the default value ?

File: docs/Configuration.md (hunk 1 above)
Comment (gaetanars) [reply docs/Configuration.md:57]: Yes but the marketplace is in the Upbound side. Crossplane is the open source project.

File: docs/Configuration.md (hunk 2 above)
Comment (gaetanars) [reply docs/Configuration.md:79]: it will may be more cleaner. I will work on it.

File: docs/Configuration.md (hunk 1 above)
Comment (azrod) [reply docs/Configuration.md:57]: it is therefore necessary to plan to replace all namespace in examples

Comment (azrod) [reply docs/Configuration.md:57]: I'm created issue to reference this : #90

File: docs/Configuration.md (hunk 2 above)
Comment (gaetanars) [reply docs/Configuration.md:79]: It looks more clear now.

**Review by azrod — APPROVED** [review]

**Pull request merged**